boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst

```diff
@@ -0,0 +1,32 @@
+.. _fanstel_ev832x:
+
+Fanstel EV-BT832X
+#################
+
+Overview
+********
+
+BluNor BT832X is a powerful, highly flexible, Bluetooth Low Energy (BLE) using
+Nordic nRF52832 SoC. With an ARM Cortex (TM) M4F MCU, available 512KB flash,
+64KB RAM, embedded 2.4GHz multi-protocol transceiver, power amplifier, and an
+integrated PCB trace antenna, or an u.FL connector for external antenna. It
+allows faster time to market with reduced development cost.
+
+For applications needing limited number of IO pins, prototyping and production
```

Comment:
I'd delete these next two paragraphs and let folks reference the material you've linked to for more information.

But you do need information about how to build a Zephyr app for this board and how to run it (check out another board document for an example).